internal/test/testdata/compile/missing_schemas.yaml

```diff
@@ -4,15 +4,15 @@ wantErrors:
   - file: resource_policies/leave_request_20210210.yaml
     error: invalid schema
     description: |-
-      Failed to load principal schema "cerbos:///non_existent_principal.json": schema "cerbos:///non_existent_principal.json" does not exist in the store
+      Failed to load principal schema "cerbos:///non_existent_principal.json": schema file "non_existent_principal.json" does not exist in the store
     position:
       line: 10
       column: 7
       path: "$.resourcePolicy.schemas.principalSchema.ref"
   - file: resource_policies/leave_request_20210210.yaml
     error: invalid schema
     description: |-
-      Failed to load resource schema "cerbos:///non_existent_leave_request.json": schema "cerbos:///non_existent_leave_request.json" does not exist in the store
+      Failed to load resource schema "cerbos:///non_existent_leave_request.json": schema file "non_existent_leave_request.json" does not exist in the store
```

Comment:
I like chomping off the `cerbos://` scheme, but I'd be tempted to prepend `_schemas/` so that it has the full path that was expected (otherwise I can see people trying to put schemas in the root of the policies directory).